### DISCUSSION TITLE
Which Windows versions are supported by WSL2 1.0.0?

### DISCUSSION THREAD

**Discussion by ValentinTwin1206** [microsoft/WSL#9232] [General]
Hi all,

which versions of Windows are supported by 1.0.0? I have read that this so called "Store version" of WSL2 only runs on Windows 11?!
Does it support Windows 10 Enterprise editions, too??
And which versions of Windows server are supported?

My company highly appreciate the `systemd` support of 1.0.0. Currently, we are using the "Windows-in" version of WSL2 which does not work with `systemd` as PID1 out of the box.

Best Valentin

**Comment by Biswa96** [comment]
> which versions of Windows are supported by 1.0.0? 

This blog pots may help https://devblogs.microsoft.com/commandline/the-windows-subsystem-for-linux-in-the-microsoft-store-is-now-generally-available-on-windows-10-and-11/

**Comment by ValentinTwin1206** [comment]
Hi @Biswa96 thanks for the quick reply. 
The blog post is definitely helping but either Windows Server or Windows Enterprise Editions are explicitly mentioned there.
Are there any plans to support older versions of Windows 10?

Best Valentin
